examples/basics/scene/image.py

```diff
@@ -20,12 +20,41 @@
 view = canvas.central_widget.add_view()
 
 # Create the image
-img_data = np.random.normal(size=(100, 100, 3), loc=128,
-                            scale=50).astype(np.ubyte)
-image = scene.visuals.Image(img_data, parent=view.scene)
+# img_data = np.random.normal(size=(100, 100, 3), loc=128,
+#                            scale=50).astype(np.ubyte)
+img_data = np.zeros(25).reshape((5, 5)).astype(np.float32)
+img_data[1:4, 1::2] = 0.5
+img_data[1::2, 2] = 0.5
+img_data[2, 2] = 1.0
+interpolation = 'nearest'
+act = 0
+image = scene.visuals.Image(img_data, interpolation=interpolation,
+                            parent=view.scene)
+canvas.title = 'Spatial Filtering using %s Filter' % interpolation
 
 # Set 2D camera (the camera will scale to the contents in the scene)
 view.camera = scene.PanZoomCamera(aspect=1)
+view.camera.set_range()
+
+# hack for now, should rename "names" to something useful
+# and make public
+names = image._names
```

Comment:
I thought about doing exactly what you suggested, but then we would miss 'nearest' and 'linear'. I added the two to the names-tuple inside ImageVisual and also to the Function-dictionary.